style.css

```diff
@@ -0,0 +1,136 @@
+body{
+    font-family: "Roboto" , sans-serif;
+    background: #ffffff;
+    width: 100%;
+    height: 100vh;
+    margin: 0;
+    color: #000000;
+    font-size: 15px;
+}
+.container {
+    width: 50%;
+    margin: 0 auto;
+    padding: 10px;
+}
+.items {
+    padding: 8px 48px;
```

Comment:
Do you really need that big number of padding? Usually, when you have a big number, it means something wrong (Not always though)